src/apps/experimental/routes/user/display/LibraryPreferences.tsx

```diff
@@ -0,0 +1,101 @@
+            <Typography variant='h2'>{globalize.translate('HeaderLibraries')}</Typography>
+
+            <FormControl fullWidth>
+                <TextField
+                    aria-describedby='display-settings-lib-pagesize-description'
+                    inputProps={{
+                        type: 'number',
+                        inputMode: 'numeric',
+                        max: '1000',
+                        min: '0',
+                        pattern: '[0-9]',
+                        required: true,
+                        step: '1'
+                    }}
+                    value={values.libraryPageSize}
+                    label={globalize.translate('LabelLibraryPageSize')}
+                    name='libraryPageSize'
+                    onChange={onChange}
+                />
+                <FormHelperText id='display-settings-lib-pagesize-description'>
+                    {globalize.translate('LabelLibraryPageSizeHelp')}
+                </FormHelperText>
+            </FormControl>
+
+            <FormControl fullWidth>
+                <FormControlLabel
+                    aria-describedby='display-settings-lib-backdrops-description'
+                    control={<Checkbox defaultChecked={values.enableLibraryBackdrops} />}
+                    label={globalize.translate('Backdrops')}
+                    name='enableLibraryBackdrops'
+                    onChange={onChange}
+                />
+                <FormHelperText id='display-settings-lib-backdrops-description'>
+                    {globalize.translate('EnableBackdropsHelp')}
+                </FormHelperText>
+            </FormControl>
+
+            <FormControl fullWidth>
+                <FormControlLabel
+                    aria-describedby='display-settings-lib-theme-songs-description'
+                    control={<Checkbox defaultChecked={values.enableLibraryThemeSongs} />}
+                    label={globalize.translate('ThemeSongs')}
+                    name='enableLibraryThemeSongs'
+                    onChange={onChange}
+                />
+                <FormHelperText id='display-settings-lib-theme-songs-description'>
+                    {globalize.translate('EnableThemeSongsHelp')}
+                </FormHelperText>
+            </FormControl>
+
+            <FormControl fullWidth>
+                <FormControlLabel
+                    aria-describedby='display-settings-lib-theme-videos-description'
+                    control={<Checkbox defaultChecked={values.enableLibraryThemeVideos} />}
+                    label={globalize.translate('ThemeVideos')}
+                    name='enableLibraryThemeVideos'
+                    onChange={onChange}
+                />
+                <FormHelperText id='display-settings-lib-theme-videos-description'>
+                    {globalize.translate('EnableThemeVideosHelp')}
+                </FormHelperText>
+            </FormControl>
+
+            { !browser.tizen && !browser.web0s && (
```

Comment:
This logic is migrated over directly from the legacy menu builder, but if anyone has context on why tizen and webOS don't have the "display missing episodes" option I'd love to add a doc comment here with a short explanation.

Reply:
IMO we should remove browser checks and make them feature flags. The wrappers for jellyfin-web can already specify which features they do and do not support (https://github.com/jellyfin/jellyfin-android/blob/master/app/src/main/assets/native/nativeshell.js)

Reply:
Let's just remove it for now. I have no idea why this check exists.

Reply:
> IMO we should remove browser checks and make them feature flags

Totally onboard with approaching this type of thing rather than explicit browser checks. Is there documentation anywhere or an SDK that clients which wrap the web client use? I started down the road of trying to understand what's there for feature flags currently, but didn't find much for context on the purpose of each flag and what they're actually controlling.

For now I removed this check following @thornbill's advice.